src/buttons/ButtonGroup.tsx

```diff
@@ -220,7 +220,7 @@ const styles = StyleSheet.create({
   },
   verticalContainer: {
     flexDirection: 'column',
-    height: null,
+    height: undefined,
```

Comment:
This actually changes the behavior of the component. We should either turn off the null check in the tsconfig or find a different solution.

Reply:
I tried in demo app, it was working fine, but if it really results in some weired behaviour we can add  `"strictNullChecks": false`  in the end

Reply:
Yeah, it might just be a corner case, but I do know that `width` and `height` being set to `null` have a special meaning apart from `undefined`. At least for TS changes, it safest to try not to change any of the JS code. There might be a Stackoverflow though that talks about how to handle this situation.

Reply:
I will check for best solution